app/src/main/java/ch/heigvd/pro/b04/android/Poll/Question/QuestionViewModel.java

```diff
@@ -0,0 +1,37 @@
+package ch.heigvd.pro.b04.android.Poll.Question;
+
+import androidx.lifecycle.MutableLiveData;
+import androidx.lifecycle.ViewModel;
+
+import java.util.LinkedList;
+import java.util.List;
+
+import ch.heigvd.pro.b04.android.Poll.Answer.Answer;
+
+public class QuestionViewModel extends ViewModel {
+    private String question;
+    private MutableLiveData<List<Answer>> answers = new MutableLiveData<>(new LinkedList<>());
+    private MutableLiveData<List<Answer>> selected = new MutableLiveData<>(new LinkedList<>());
```

Comment:
I don't understand why we need a `selected` list if we know which answer was clicked on (because it triggers a method) and if the answer was selected or deselected (via it's attribute).